.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
+## Information
+
+### Swift version
+
+What does this command give you?
+```
+$ swift --version
+```
+
+### Operating system
+
+What does this command give you?
+```
+$ uname -a
+```
+
+### Driver version
+
+What does this command give you?
+```
+$ cat Package.swift # Only applies if you are using Swift package manager
+```
+*or*
+```
+$ cat Podfile # Only applies if you are using Cocoapods
+```
+
+### What is the version(s) of `mongod` that you are running with the driver?
+Try running:
+```
+$ mongod --version
+```
+or, running this in a MongoDB shell connected to the relevant node(s):
+```
+> db.version()
+```
+
+### What is your server topology?
+How is your MongoDB deployment configured?
```

Comment:
Did you mean to have line breaks between the title and paragraph text here and in the "How did you install..." question below?

Reply:
Yep, you're right. I've also added more whitespace to separate the each question from each other.

Reply:
(It does not affect the rendering from what I can see).